mappings/muse.py

```diff
@@ -1,100 +1,170 @@
-            'contributors': [
-                ('260', '{f}|||manufacturer'),
-                ('700', '{a} {b} {c} {d}|||{e}'),
-                ('710', '{a} {b} {c} {d}|||{e}'),
-                ('711', '{a} {e}|||{j}'),
-            ],
-            'is_part_of': ('490', '{a}|{v}|volume'),
-            'has_part': [
-                ('856', '1|{u}|muse|text/html|{{"reader": false, "download": false, "catalog": false, "embed": true}}')
-            ]
-        }
+class MUSEMapping(BaseMapping):
+    def __init__(self, muse_record):
+        self.record = self.map_muse_record(muse_record)
+
+    def map_muse_record(self, muse_record) -> Record:
+        identifiers = self._get_identifiers(muse_record)
+        alternative = self._get_formatted_field(muse_record, '240', ['a', 'k'], '{} {}')
+        has_version = self._get_formatted_field(muse_record, '250', ['a', 'b'], '{}|')
+        spatial = self._get_formatted_field(muse_record, '264', ['a'], '{}')
+        extent = self._get_formatted_field(muse_record, '300', ['a', 'b', 'c'], '{}{}')
+        toc = self._get_formatted_field(muse_record, '505', ['a'], '{}')
+
+        rights = 'muse|https://creativecommons.org/licenses/by-nc/4.0/||Creative Commons Attribution-NonCommercial 4.0 International|'
+
+        new_record = Record(
+            uuid=uuid4(),
+            frbr_status=FRBRStatus.TODO.value,
+            cluster_status=False,
+            source=Source.MUSE.value,
+            source_id=list(identifiers[0].split('|'))[0],
+            identifiers=identifiers,
+            authors=self._get_authors(muse_record),
+            title=self._get_title(muse_record),
+            alternative=alternative[0] if alternative else None,
+            has_version=has_version[0] if has_version else None,
+            publisher=self._get_publishers(muse_record),
+            spatial=spatial[0] if spatial else None,
+            dates=self._get_dates(muse_record),
+            languages=self._get_languages(muse_record),
+            extent=extent[0] if extent else None,
+            table_of_contents=toc[0] if toc else  None,
+            abstract=self._get_abstracts(muse_record),
+            subjects=self._get_subjects(muse_record),
+            contributors=self._get_contributors(muse_record),
+            is_part_of=self._get_formatted_field(muse_record, '490', ['a','v'], '{}|{}|volume'),
+            has_part=self._get_formatted_field(muse_record, '856', ['u'], '1|{}|muse|text/html|{{"reader": false, "download": false, "catalog": false, "embed": true}}'),
+            rights=rights,
+            date_created=datetime.now(timezone.utc).replace(tzinfo=None),
+            date_modified=datetime.now(timezone.utc).replace(tzinfo=None)
+        )
 
-    def applyFormatting(self):
-        self.record.source = 'muse'
-        self.record.source_id = list(self.record.identifiers[0].split('|'))[0]
+        return new_record
+
+    def _get_formatted_field(self, record, field, subfields, string_format):
```

Comment:
What are your thoughts on adding the subfields in the string_format? 

That could simplify lines, e.g. 

```
self._get_formatted_field(record, '100', ['a', 'b', 'c', 'd'], '{} {} {} {}|||true'),
```
to
```
self._get_formatted_field(record, '100', '{a} {b} {c} {d}|||true'),
```

When we get into these more formatted strings, python's way of formatting a dict comes in handy.